### DISCUSSION TITLE
Welcome to Vite-React-dApp-Template Discussions!

### DISCUSSION THREAD

**Discussion by huseyindeniz** [huseyindeniz/vite-react-dapp-template#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oehm-smith** [comment]
Hi @huseyindeniz , I would like to open discussions about Wallets.

1. It looks like ethers6 has clobbered ethers5 after you made the change to introduce 6.  Given they are quite different it is good to have both versions.

Currently the `package.json` has:

```
  "dependencies": {
    ...
    "ethers": "^6.10.0",
    ...
```

But you might want to do something like this for npm (I'm not sure of the exact syntax for Yarn):

```
npm install ethers5@npm:ethers@5
npm install ethers6@npm:ethers@6
```

**Comment by huseyindeniz** [comment]
hi @oehm-smith ,
In my dapps, I delete ethersV5 folder completely and not use it anymore. We could add both versions like you suggested but it's better to delete ethersV5 completely I think. What do you think? Do you prefer using ethersV5? 

Btw, I abstracted the underlying blockchain library package for supporting [Viem](https://github.com/huseyindeniz/vite-react-dapp-template/issues/5). I'm not fan of Viem but [TypeChain](https://github.com/dethcrypto/TypeChain) is abandoned by it's creator and according to Viem community, Viem doesn't need TypeChain. Anyway, what's in my mind is smt like this in the end:

-services
-- ethers <- without version(latest version)
-- viem

And I'll keep using ethers in my dapps.
